### PR TITLE
Update modal window

### PR DESCRIPTION
**Pull request by andyvesel**
Updated modal window

- Added buttons
- Added text fields

### CODE REVIEW

**Comment by andyvesel** [comment]
@bond95 should that be like `src/app/cluster/modals/AddClusterModal` thing?

**Comment by bond95** [comment]
> @bond95 should that be like src/app/cluster/modals/AddClusterModal thing? Or modals directory is unneeded?

Modal directory is needed only in case if there will be more than one modals for clusters.

**Comment by andyvesel** [comment]
@bond95 fixed

File: src/app/cluster/AddClusterModal.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import {
+  Modal,
+  Button,
+  TextInput,
+  TextContent,
+  TextList,
+  TextListItem,
+  TextArea,
+} from '@patternfly/react-core';
+import React from 'react';
```
Comment (bond95) [inline src/app/cluster/AddClusterModal.tsx]: Please, move this import to first line, to keep consistency.

File: src/app/cluster/AddClusterModal.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+const AddClusterModal = ({ onHandleModalToggle, isModalOpen, ...props }) => (
```
Comment (bond95) [inline src/app/cluster/AddClusterModal.tsx]: Please, add type check.

File: src/app/cluster/AddClusterModal.tsx (hunk 1 above)
Comment (eriknelson) [reply src/app/cluster/AddClusterModal.tsx]: We need to decide if we care about import ordering and if so, codify that in the linter.

Comment (bond95) [reply src/app/cluster/AddClusterModal.tsx]: I don't so much care about import ordering, but React import first is something like `unspoken rule`. If you look at code in other React projects or at least at tutorial React import in most cases is in first place.

Comment (andyvesel) [reply src/app/cluster/AddClusterModal.tsx]: Ok, fixed

File: src/app/cluster/AddClusterModal.tsx (hunk 2 above)
Comment (andyvesel) [reply src/app/cluster/AddClusterModal.tsx]: Added

**Comment by eriknelson** [comment]
@andyvesel needs rebase

**Comment by andyvesel** [comment]
@eriknelson done

**Pull request merged**